### DISCUSSION TITLE
Testnet solution suggestions for projects to be built on IBAX

### DISCUSSION THREAD

**Discussion by Tubikali** [IBAX-io/go-ibax#1950] [Ideas] [answered]
Hello,

Investors who will develop projects in the ibax ecosystem may want to establish a testnet to create a community and prevent system disruptions. Is there a way to follow for this?

**Comment by yolandadadada** [comment, marked as answer]
1. You can build your own test environment for testing
2. You can test directly in the IBAX test network
